new-site/src/docs/elements/components/selection-group/accessibility.mdx

```diff
@@ -0,0 +1,13 @@
+---
+slug: '/elements/components/selection-group/accessibility'
+title: 'SelectionGroup - Accessibility'
+---
+
+import { Link } from 'hds-react';
+
+## Accessibility
+
+### Pay attention to
+- Aim to make the order of Selection group options as intuitive as possible for assistive technologies. This depends on the group content. An alphabetical ordering is a good starting point, but for some content it can be possible to order them by size (smaller to larger), etc.
```

Comment:
Not sure if it's good to use list if we only have one item there?

Reply:
I see a benefit in keeping it consistent across all documentation pages. Otherwise, it may be hard to find this information if it is in the middle of some other page.

Also, we will probably add more accessibility content later on.